astropy/coordinates/solar_system.py

```diff
@@ -0,0 +1,271 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+This module contains convenience functions for retrieving solar system
+ephemerides from jplephem.
+"""
+
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+import numpy as np
+from ..extern import six
+from .sky_coordinate import SkyCoord
+from ..utils.data import download_file
+from .. import units as u
+from ..constants import c as speed_of_light
+from .representation import CartesianRepresentation
+from .builtin_frames import GCRS, ICRS
+from .builtin_frames.utils import get_jd12, cartrepr_from_matmul
+from .. import _erfa
+
+__all__ = ["get_body", "get_moon", "get_barycentric_body_position", "SOLAR_SYSTEM_BODIES"]
+
+KERNEL = None
+
+"""
+each value in the BODIES dictionary a list of kernel pairs needed
+to find the barycentric position of that object from the JPL kernel.
+"""
+SOLAR_SYSTEM_BODIES = ['sun', 'mercury', 'venus', 'earth-moon-barycenter',
```

Comment:
I decided against this because making ```BODIES``` public exposes the nasty "kernel chain" entries to the general user. Keeping ```BODIES``` hidden, but making a list of valid keys gives a cleaner API, in my opinion.